brpc-java-core/src/main/java/com/baidu/brpc/protocol/http/HttpRpcProtocol.java

```diff
@@ -177,7 +177,7 @@ public Object decode(ChannelHandlerContext ctx, DynamicCompositeByteBuf in, bool
     @Override
     public ByteBuf encodeRequest(Request request) throws Exception {
         HttpRequest httpRequest = (HttpRequest) request;
-        String serviceName = httpRequest.getTargetMethod().getDeclaringClass().getSimpleName();
+        String serviceName = httpRequest.getTargetMethod().getDeclaringClass().getName();
```

Comment:
if user use BrpcMeta, it should use the serviceName of BrpcMeta

Reply:
if user use  BrpcMeta , line 184  will set the serviceName of BrpcMeta

Reply:
@JoeCqupt OK. I don't see this code originally.